source/_integrations/opentherm_gw.markdown

```diff
@@ -52,6 +52,10 @@ id:
 Please make sure no other device or application is connected to the OpenTherm Gateway at the same time as Home Assistant. This is not a supported scenario and may lead to unexpected results.
 </div>
 
+<div class='note info'>
+Once integration is set up, only `climate` entity is enabled. To avoid clutter in the state history, all sensors are disabled by default. If you're willing to use them, please navigate to the entities list and enable them. Not all the entities will be populated as different boilers support different features. 
```

Comment:
The disabled entities are already documented here:
https://github.com/home-assistant/home-assistant.io/blob/b178319336b29e988c61b058be6646c80f563eaf/source/_integrations/opentherm_gw.markdown?plain=1#L30

The lack of support for all features is already documented here:
https://github.com/home-assistant/home-assistant.io/blob/b178319336b29e988c61b058be6646c80f563eaf/source/_integrations/opentherm_gw.markdown?plain=1#L222
and here:
https://github.com/home-assistant/home-assistant.io/blob/b178319336b29e988c61b058be6646c80f563eaf/source/_integrations/opentherm_gw.markdown?plain=1#L422